### PR TITLE
Improved urls docs

### PR DESCRIPTION
**Pull request by evildmp**
### Summary

* clarified wording and examples
* created a new how-to section for handling multilingual content
* moved content as appropriate

[ci only docs]

### CODE REVIEW

File: docs/how_to/index.rst [hunk 1]
```diff
@@ -8,21 +8,22 @@ These guides presuppose some familiarity with django CMS. They cover some of
...
-    sitemap
-    page_types
-    wizards
-    contributing
+    Install django CMS by hand <install>
+    Create custom plugins <custom_plugins>
+    Customise menus <menus>
+    Create apphooks <apphooks>
+    Manage complex apphook configuration <namespaced_apphooks>
+	Serve multiple languages <languages>
```
Comment (czpython) [inline docs/how_to/index.rst]: indentation?

File: docs/how_to/languages.rst [hunk 2]
```diff
@@ -0,0 +1,123 @@
+.. _multilingual_support_how_to:
+
+###############################
+How to serve multiple languages
+###############################
+
+If you used the `django CMS installer <https://github.com/nephila/djangocms-installer>`_ to start your project, you'll find
+that it's already set up for serving multilingual content. Our :ref:`installation` guide also does the same.
+
+Thiss guide specifically describes the steps required to set up multilingual support.
```
Comment (czpython) [inline docs/how_to/languages.rst]: This

Comment (czpython) [inline docs/how_to/languages.rst]: I think "specifically" is not necessary here.
"This guide describes the steps.."

File: docs/how_to/languages.rst [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+
+    {% page_language_url "de" %}
+
+
+***************************************
+Configuring language-handling behaviour
+***************************************
+
+:setting:`CMS_LANGUAGES` describes the all options available for determining how django CMS serves content across multiple
+languages .
```
Comment (czpython) [inline docs/how_to/languages.rst]: extra space in "languages."

File: docs/spelling_wordlist [hunk 4]
```diff
@@ -240,6 +240,4 @@ YouTube
 Zürich
 Øyvind
 日本語
-
-
-
+addes
```
Comment (czpython) [inline docs/spelling_wordlist]: Shouldn't it be added?

File: docs/topics/i18n.rst [hunk 5]
```diff
@@ -2,138 +2,38 @@
...
-*****************
+See :ref:`multilingual_support_how_to` on how to set up a multilingual django CMS project.
 
-If you use more than one language, django CMS urls, *including the admin URLS*, need to be
-referenced via :func:`~django.conf.urls.i18n.i18n_patterns`. For more information about this see
+****
+URLs
+****
+
+Multilingual URLs require the use of :func:`~django.conf.urls.i18n.i18n_patterns`. See For more information about this see
```
Comment (czpython) [inline docs/topics/i18n.rst]: "See For more information about this see" should be "For more information about this see"

**Comment by evildmp** [comment]
@czpython please take another look

**Pull request merged**